libconway/src/grids.rs

```diff
@@ -199,6 +199,22 @@ impl BitGrid {
             }
         }
     }
+
+    /// Calls callback on each bit that is set (1). Callback receives (col, row).
+    pub fn each_set<F: FnMut(usize, usize)>(&self, mut callback: F) {
+        for row in 0 .. self.height() {
+            let mut col = 0;
+            for col_idx in 0 .. self.width_in_words() {
+                for shift in (0..64).rev() {
```

Comment:
`64` is a magic number. I know it won't change any time soon but giving it a name helps with *the why*. This is used all over libconway code.

Reply:
What do you think about adding a `bits!` macro that will count the bits in the given type (at compile time). Then the `64` could be replaced with `bits!(u64)`.

It can be done at compile time because both `max_value` and `count_ones` are const fns:
```
$type::max_value().count_ones() as usize
```

Reply:
Great suggestion! Especially the compile-time aspect. :vulcan_salute: